src/Servers/Kestrel/samples/Http3SampleApp/Program.cs

```diff
@@ -27,7 +27,6 @@ public static void Main(string[] args)
                     webHost.UseKestrel()
                     .UseQuic(options =>
                     {
-                        options.Certificate = cert; // Shouldn't need this either here.
                         options.Alpn = "h3-29"; // Shouldn't need to populate this as well.
```

Comment:
Can ALPN be customized on a listener today?

I left this global for now because I assume it will be removed once HTTP/3 is finalized.

Reply:
ALPN can be customized in one of the callbacks that lets you specify the SslServerAuthenticationOptions.
https://github.com/dotnet/aspnetcore/blob/b86257101783524b0bad3287dbb50af869a3db56/src/Servers/Kestrel/Core/src/HttpsConnectionAdapterOptions.cs#L93

Reply:
Yeah, I believe so: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs#L169.

I believe we can set the ALPN here to the right value.

Reply:
We currently pass that type down to msquic as well: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicListener.cs#L31. Again probably just some work to expose it correctly.